README.md

```diff
@@ -193,6 +193,29 @@ performance.
 Hot reloading code is just one line in the beginning and one line at the end of
 each module so you might not need source maps at all.
 
+## Preact
+
+React-hot-loader should work out of the box with `preact-compact`, but, in case of pure preact, you will need
+to configure it:
+
+* create configuration file (setupHotLoader.js)
+
+```js
+import reactHotLoader from 'react-hot-loader'
+import preact from 'preact'
+
+reactHotLoader.inject(preact, 'h') // inject RHL into `h` on `preact`.
```

Comment:
Not sure about exposing this command, but nothing will work without it (except idea 1 in the source issue)